src/countOccurrences.js

```diff
@@ -19,6 +19,14 @@
  */
 function countOccurrences(phrase, letter) {
   // write code here
+  const str = phrase.toLowerCase();
+  let total = '';
```

Comment:
new line before loop or function ...
```suggestion
  let total = '';
  
```

Reply:
I prefer `counter`